### PR TITLE
Fixes #12824: In directive, change input for parameter from technique editor from text to textarea 

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/12824

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Commit modified

File: rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala [hunk 1]
```diff
@@ -138,6 +138,10 @@ class TechniqueWriter (
       <INPUT>
         <NAME>{parameter.id.value.toUpperCase()}</NAME>
         <DESCRIPTION>{parameter.name.value}</DESCRIPTION>
+        <CONSTRAINT>
+          <TYPE>textarea</TYPE>
+          <MAYBEEMPTY>true</MAYBEEMPTY>
```
Comment (ncharles) [inline rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala]: are you sure it can be empty ?

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**